packages/ckeditor5-table/src/tablecolumnresize/tablecolumnresizeutils.ts

```diff
@@ -0,0 +1,517 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module table/tablecolumnresize/tablecolumnresizeutils
+ */
+
+import { isEqual } from 'lodash-es';
+
+import { Plugin, type Editor } from 'ckeditor5/src/core.js';
+
+import type { DowncastWriter, Element, ViewElement, ViewNode } from 'ckeditor5/src/engine.js';
+import TableUtils from '../tableutils.js';
+
+import { COLUMN_MIN_WIDTH_IN_PIXELS } from './constants.js';
+import {
+	clamp, getColumnEdgesIndexes, getColumnGroupElement, getDomCellOuterWidth,
+	getElementWidthInPixels,
+	getTableColumnsWidths, getTableWidthInPixels, sumArray, toPrecision
+} from './utils.js';
+
+import TableWalker from '../tablewalker.js';
+
+/**
+ * The table column resize editing plugin.
+ */
+export default class TableColumnResizeUtils extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	public static get requires() {
+		return [ TableUtils ] as const;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public static get pluginName() {
+		return 'TableColumnResizeUtils' as const;
+	}
+
+	/**
+	 * Performs resize of specified by resizer column with specified width.
+	 * It composes steps from {@link #_prepareColumnResize}, {@link #_assignColumnWidth} and {@link #_endResize}.
+	 *
+	 * @internal
+	 * @param resizerElement Resize column handle view element.
+	 * @param newColumnWidth New column width in pixels.
+	 * @param compensateWidthWhenMoveCenteredTable
+	 */
+	public _resizeColumnUsingResizer(
```

Comment:
`@internal` flag and `_` at the name's start are not necessary since this meant to be a public helper. The same applies to all the other methods in the plugin.

Reply:
@Dumluregn I used the same naming convention as it is in the core and many other places:
https://github.com/ckeditor/ckeditor5/blob/ck/6023/packages/ckeditor5-engine/src/model/element.ts#L219-L252 
https://github.com/ckeditor/ckeditor5/blob/08270aafcc6e7e4fb4f4a4d2ebbc7451f031d303/packages/ckeditor5-engine/src/model/position.ts#L877-L901

Are you sure?

Reply:
@Dumluregn done